linkage/pvs_like_case_study/generate_simulated_data/generate_simulated_data_small_sample.py

```diff
@@ -137,32 +150,18 @@ def dedupe_preserving_source_records(df, columns_to_dedupe, source_records_col='
 w2_1099_ground_truth = w2_1099.set_index('record_id').simulant_id
 w2_1099 = w2_1099.drop(columns=['simulant_id'])
     
+taxes = pd.concat([taxes_1040, w2_1099], ignore_index=True)
+    
+taxes_ground_truth = pd.concat([taxes_1040_ground_truth, w2_1099_ground_truth])
+    
 # "IRS records do not contain DOB, and many of the records contain only the first four letters of the last name."
 # (Brown et al. 2023, p.30, footnote 19)
 # This should be updated in pseudopeople but for now we do it here.
```

Comment:
Now that it has been updated in psp you can update this comments.

Reply:
The second part of the quote (truncating last name) is still not done in pseudopeople. The first part (DOB not being in IRS records) was the only update. I've minimally changed this comment to reflect that.